src/components/BreadcrumbsItem/BreadcrumbsItem.tsx

```diff
@@ -26,22 +39,92 @@ export interface Props {
  * import {BreadcrumbsItem} from "@chanzuckerberg/eds";
  * ```
  *
- * A single breadcrumb, to be used in the Breadcrumbs component.
+ * A single breadcrumb subcomponent, to be used in the Breadcrumbs component.
  */
-export const BreadcrumbsItem = ({ className, text, href, ...other }: Props) => {
-  const componentClassName = clsx(styles['breadcrumbs__item'], className, {});
+export const BreadcrumbsItem = ({
+  dropdownMenuItems,
+  className,
+  href,
+  text,
+  variant,
+  ...other
+}: Props) => {
+  const [isActive, setIsActive] = React.useState(false);
+
+  /**
+   * Closes the dropdown menu if there is focus or click outside.
+   */
+  const handleBlur = (event: React.FocusEvent<HTMLElement, Element>) => {
+    if (!event.currentTarget.contains(event.relatedTarget) && isActive) {
+      setIsActive(false);
+    }
+  };
+
+  const componentClassName = clsx(
+    styles['breadcrumbs__item'],
+    variant === 'back' && styles['breadcrumbs__item-back'],
+    className,
+  );
+
+  const ellipsisButtonClassName = clsx(
+    styles['breadcrumbs__link'],
+    styles['breadcrumbs__ellipsis'],
+  );
+
+  const dropdownMenuClassName = clsx(
+    styles['breadcrumbs__dropdown-menu'],
+    isActive && styles['breadcrumbs__dropdown-menu--active'],
+  );
+
   return (
     <li className={componentClassName} {...other}>
-      <a className={styles['breadcrumbs__link']} href={href}>
-        {text}
-      </a>
-      <Icon
-        className={styles['breadcrumbs__icon']}
-        color={EdsThemeColorIconNeutralStrong}
-        name="chevron-right"
-        purpose="decorative"
-        size="0.75rem"
-      />
+      {/**
+       * 1) The collapsed variant is a button with ellipsis. Interaction spawns a dropdown containing the collapsed breadcrumb links.
+       * 2) The back variant is a left pointing icon that usually links to the second last breadcrumb href.
+       * 3) The default variant displays the prop text and links the prop href.
+       * 4) The decorative separator between breadcrumbs items.
+       */}
+      {
+        /* 1 */
+        variant === 'collapsed' ? (
+          <>
+            <button
+              className={ellipsisButtonClassName}
+              onClick={() => {
+                setIsActive(!isActive);
+              }}
+            >
+              ...
+            </button>
+            <DropdownMenu
+              className={dropdownMenuClassName}
+              isActive={isActive}
+              onBlur={handleBlur}
+            >
+              {dropdownMenuItems}
+            </DropdownMenu>
+          </>
+        ) : (
+          <a className={styles['breadcrumbs__link']} href={href as string}>
+            {/* 2 */}
+            {variant === 'back' ? (
+              <Icon
+                className={styles['breadcrumbs__back-icon']}
+                name="chevron-left"
+                purpose="informative"
+                title="back"
```

Comment:
Not sure if this is the appropriate title

Reply:
Should the accessible title be the `text` then? Since that's where the link is going, it sounds like.

Reply:
Yes! Will change